src/web/mjs/connectors/kuaikanmanhua.mjs

```diff
@@ -81,15 +77,9 @@ export default class Kauikanmanhua extends Connector {
     async _getPageList(manga, chapter, callback) {
         try {
             let request = new Request(this.url + chapter.id, this.requestOptions);
-            let data = await this.fetchDOM(request, 'div.entry-content noscript');
-            let copy = [];
-            data.forEach(function (item) {
-                //create new variable to work with
-                let str = item.innerText;
-                //push only the url of "src" into the copy array
-                copy.push(str.substring(str.lastIndexOf("src='") + 5, str.lastIndexOf("g'") + 1));
-            });
-            let pageList = copy.map(element => this.getAbsolutePath(element, request.url));
+            let data = await this.fetchDOM(request, 'div.imgList source');
```

Comment:
Try fetchUI instead to get it from a headless browser window

```javascript
let script = `
new Promise(resolve => {
    let pages = __NUXT__.data[0].comicInfo.comicImages.map(img => img.url);
    resolve(pages);
});
`;
let request = new Request(this.url + chapter.id, this.requestOptions);
let pageList = await Engine.Request.fetchUI(request, script);
callback(null, pageList);
```

Reply:
I'll look into it, thanks for the tip!